# Header

geometry_msgs/Pose2D[] pathpoints

int32[] target_points

string[] target_names
